src/api/components/SubmitForm.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { USER_ID } from '../todos';
+interface Props {
+  inputClassName: string;
+
+  todos?: Todo[];
+  onAddTodo?: (todo: Todo) => void;
+
+  updateTodo?: Todo;
+  setIsUpdate?: (val: boolean) => void;
+  OnUpdateTodo?: (todo: Todo) => void;
+  OnDelete?: (id: number) => void;
```

Comment:
```suggestion
  onUpdateTodo?: (todo: Todo) => void;
  onDelete?: (id: number) => void;
```
only class start with uppercase